kouncil-frontend/apps/kouncil/src/app/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,68 @@
+import {ChangeDetectionStrategy, Component, HostBinding} from '@angular/core';
+import {Observable} from 'rxjs';
+import {AuthService, KouncilRole} from '@app/common-auth';
+import {SidebarService} from './sidebar.service';
+
+@Component({
+  selector: 'app-sidebar',
+  template: `
+    <div class="sidenav" [ngClass]="(currentState$ | async) ? 'opened' : 'closed'">
+
+      <app-sidebar-menu-item [label]="'Topics'" [icon]="'topic'" [routeLink]="'/topics'"
+                             *ngIf="(isAuthenticated$ | async) && authService.canAccess([KouncilRole.KOUNCIL_EDITOR, KouncilRole.KOUNCIL_VIEWER])">
+      </app-sidebar-menu-item>
+
+      <app-sidebar-menu-item [label]="'Brokers'" [icon]="'hub'" [routeLink]="'/brokers'"
+                             *ngIf="(isAuthenticated$ | async) && authService.canAccess([KouncilRole.KOUNCIL_ADMIN])">
+      </app-sidebar-menu-item>
+
+      <app-sidebar-menu-item [label]="'Consumer Groups'" [icon]="'device_hub'"
+                             [routeLink]="'/consumer-groups'"
+                             *ngIf="(isAuthenticated$ | async) && authService.canAccess([KouncilRole.KOUNCIL_ADMIN])">
+      </app-sidebar-menu-item>
+
+      <app-sidebar-menu-item [label]="'Track'" [icon]="'manage_search'" [routeLink]="'/track'"
+                             *ngIf="(isAuthenticated$ | async) && authService.canAccess([KouncilRole.KOUNCIL_EDITOR, KouncilRole.KOUNCIL_VIEWER])">
+      </app-sidebar-menu-item>
+
+      <app-sidebar-menu-item [label]="'Schema Registry'" [icon]="'code'" [routeLink]="'/schemas'"
+                             *ngIf="(isAuthenticated$ | async) && authService.canAccess([KouncilRole.KOUNCIL_EDITOR, KouncilRole.KOUNCIL_VIEWER])">
+      </app-sidebar-menu-item>
+
+      <div style="width: 100%; height: 32px">
+        <button mat-icon-button
+                style="bottom: 20px; right: 0; position: absolute;"
+                (click)="changeState()">
+          <mat-icon style="margin: 0" class="material-symbols-outlined">
+            {{ (currentState$ | async) ? 'dock_to_right' : 'dock_to_left' }}
+          </mat-icon>
+        </button>
+      </div>
+    </div>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./sidebar.component.scss']
+})
+export class SidebarComponent {
```

Comment:
The padding and margins of the topic list near the sidebar do not look good to me (see screenshot below). Maybe we can change the position of the "Create topic" button or just add a "+" button somewhere, so there will not be so much blank space above.
![image](https://github.com/user-attachments/assets/92a6ab56-0720-4320-b446-6917b7637b06)

Reply:
As discussed, we will think about some additional functionality that will make this bar more appealing